amundsen_application/static/js/ducks/search/reducer.ts

```diff
@@ -40,6 +47,7 @@ export function searchResource(resource: ResourceType, term: string, pageIndex:
 
 const initialState: SearchReducerState = {
   search_term: '',
+  isLoading: true,
```

Comment:
Can we default to `false`, or do you currently end up seeing the no results message followed by the spinner followed by the actual search results when it completes? 

Would that also remove the need to have the following code?:
```
 case GetPopularTables.SUCCESS:
 case GetPopularTables.FAILURE:
      return {
        ...state,
        isLoading: false,
      };
```

Reply:
just tried it out, that works and is way cleaner!